packages/client/src/v2-events/custom-api/index.ts

```diff
@@ -17,6 +17,7 @@ import { trpcClient } from '@client/v2-events/trpc'
 export interface OnDeclareParams {
```

Comment:
These are unrelated – just decided to fix as I noticed +100 `DECLARE` actions in my database after something was throwing a backend exception